src/errors/BasisTheoryReactorError.js

```diff
@@ -1,11 +1,36 @@
+// this method avoids a TypeError if the error contains a circular reference, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Cyclic_object_value
+const safeStringify = (obj) => {
+  const seen = new WeakSet();
+
+  return JSON.stringify(obj, (key, value) => {
+    if (typeof value === 'object' && value !== undefined) {
+      if (seen.has(value)) {
+        // circular reference found, return a placeholder or skip it
+        return '[Circular]';
+      }
+
+      seen.add(value);
+    }
+
+    return value;
+  });
+};
+
 const sanitizeErrors = (errors) => {
```

Comment:
should we just wrap this in a try catch and if anything throws, we throw an "expected" unexpected error?

Reply:
This will make it so Vault doesn't blow up.